src/RawPOSIX/src/interface/timer.rs

```diff
@@ -111,7 +114,7 @@ impl IntervalTimer {
                         .saturating_sub(guard.start_instant.elapsed());
 
                     if remaining_seconds == RustDuration::ZERO {
-                        lind_kill_from_id(guard.cageid, 14);
+                        lind_send_signal(guard.cageid, SIGALRM);
```

Comment:
Perhaps a short comment explaining why you're doing this (i.e. why you're using a signal at all to set up a timer) would be helpful.   I think this is largely for compatibility / interruptibility of timers, etc. but just writing 2-3 sentences somewhere here may give context.